crates/masonry/src/render_root.rs

```diff
@@ -532,14 +532,15 @@ impl RenderRoot {
 
         self.update_focus();
 
-        // If we need a new paint pass, make sure winit knows it.
-        if self.wants_animation_frame() {
+        if self.root.state().request_anim {
             self.state
                 .signal_queue
                 .push_back(RenderRootSignal::RequestAnimFrame);
         }
 
-        if self.root.state().needs_paint {
+        // If we either need to be repainted of need to update accessibility info,
```

Comment:
... or need ...

Reply:
(I'm not sure that this comment actually contains any information either)